### ISSUE TITLE
Gestion des images, quel attributs pour les images collectées sur GeoNature-citizen

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/GeoNature-citizen#54]
Pour moi ce sont tous des champs texte optionnels : 
- La légende peut être plus fine que le nom d'espèce (avec le lieu, le contexte...). 
- L'auteur du média est pas forcément le même que celui qui saisit l'observation
- Source, c'est différent du programme, c'est l'éventuelle structure à l'origine du média (en complément de l'auteur)
- Pour le licence, je ne sais pas si le portail peut l'imposer globalement, d'autant plus si le média a déjà une licence existante. 

Mais tout ça est peut-être hors cadre pour des médias illustrant une observation.

_Originally posted by @camillemonchicourt in https://github.com/PnX-SI/GeoNature-citizen/issues/3#issuecomment-467399170_

**Comment by lpofredc** [comment]
Oui, ce sont des informations importantes pour la gestion des médias. Mais cela risque de réduire la simplicité de la saisie des observations qui doit rester la plus simple possible. 

- **Titre**: avec un champ libre ou, si vide, généré automatiquement. Mais ça remets en cause le formulaire proposé dans les maquettes.
- **Auteur**: une pratique qui se fait est de demander au contributeur de certifier qu'il est bien l'auteur de l'image (une case à cocher). 
- **Source**: Pour la source, ne pourrais-t-on pas lier cela à la structure dont fait partie l'observateur > donc plus simplement avec une clé étrangère vers la table des utilisateurs ?
- **Licence**: peut-être imposer une licence en avertissant l'observateur. charge a lui de savoir si son media est compatible

Peut-être qu'une solution qui simplifierai une partie de ces questionnements serait que l'ajout de médias soit réservé aux utilisateur enregistrés.

**Comment by camillemonchicourt** [comment]
Oui c'est peut-être trop précis dans notre cas où ce sera toujours l'observateur qui chargera une photo personnelle de son observation.

Par contre ça serait dommage de limiter l'ajout de médias uniquement aux utilisateurs connectés.

**Comment by samuelpriou** [comment]
Ce sont effectivement des informations intéressantes mais cela risque de ralentir la saisie. 

Si on fait ce choix, il faut laisser la possibilité aux citoyens de saisir un titre libre en limitant le nombre de caractères. 
Pour l'auteur il faudrait peut être recourir à une automatisation en récupérant le prénom et le nom sur le compte. Cela pose la question pour ceux qui n'ont pas de compte...

Je partage l'idée de Frédéric pour la licence en imposant une licence (CC-BY) autorisant l'utilisation de la photo.
Pour la source ce n'est me semble t'il pas un besoin pour le PNM.
